spec.md

```diff
@@ -51,7 +60,62 @@ for public re-use.
                 ├── 0         # Each multiscale level is stored as a separate Zarr array, as above, but only integer values
                 │   ...       # are supported.
                 └── n
+```
+
+### High-content screening
+
+The following layout should describe the hierarchy for a high-content screening
+dataset. There are exactly four levels of hierarchies above the images:
 
+- the top-level group defines the plate, it MUST implement the plate
+  specification defined below
+- the second group defines all acquisitions performed on a single plate. If 
+  only one acquisition was performed, a single group must be used.
+- the third group defines all the well rows available for an acquisition
+- the fourth group defines all the well columns available for a given well row
+- the fifth group defined all the individual fields of views for a given well.
+  The fields of views are images, SHOULD implement the "multiscales"
+  specification, MAY implement the "omero" specification and MAY contain 
+  labels.
```

Comment:
I expect we'll need to extend this syntax to elsewhere in the spec. For example, currently we have: "Information specific to the channels of an image and how to render it can be found under the "omero" key in the group-level metadata" suggests that an Image MUST have "omero" specification.
Probably, if this was improved then we don't need to specify the image metadata requirements here (under HCS) and under Images, but can refer from here to the Images spec (unless the Image spec requirements are different when under HCS? Probably not).

Reply:
I also noticed that currently the Images spec doesn't define whether the image contains Labels? e.g. in s3 there would be no way to know whether an image has labels.

Reply:
There would be no way to find the labels that aren't in the `labels/` group, but those are discoverable.

Reply:
Yes I introduced RFC2119 as I suspect it might be useful to start thinking about these concepts both at the level of the individual keys but also at the level of the various specifications. If we agree, there will certainly be a work of reviewing the spec to introduce this terminology but a lot of this will be outside the scope of this PR.

While this terminology naturally applies to the definition of the individual keys applies naturally, I found it harder to use for the scope of each specification though. Taking your example, I don't think you MUST have an `omero` specification, but if you want some rendering metadata, you definitely SHOULD. 

> I also noticed that currently the Images spec doesn't define whether the image contains Labels? e.g. in s3 there would be no way to know whether an image has labels.

From the current spec, my understanding is that an image contains labels if the image group contains a sub-group implementing the `labels` specification?

Reply:
I guess that the discoverability of labels based on the existence of a sub-directory is different from the existence of e.g. Wells, Columns, Multiscale levels etc which are explicitly defined in the parent containers. To discover labels, you'd have to be aware of the spec to know to check, but for other sub-directories, you could find these by inspecting the parent metadata, without knowing the spec beforehand, which seems like a nice feature.